modules/vstudio/_preload.lua

```diff
@@ -265,6 +265,22 @@
 		kind = "string",
 	}
 
+	-- Directory of LLVM install
+	p.api.register {
+		name = "llvmdir",
+		scope = "config",
+		kind = "directory",
+		tokens = "true",
+	}
+
+	-- Version of LLVM Install
+	p.api.register {
+		name = "llvmversion",
```

Comment:
This is for explicit handling of custom LLVM installs. I believe LLVM and Clang versions are tied together, but I'm not 100% certain.

Reply:
To update, this would handle things like explicit version numbers, rather than just "16" as the major. You would use this to handle 16.0.5 or something like that.

Reply:
Thanks for info.